cores/esp8266/Updater.h

```diff
@@ -5,6 +5,7 @@
 #include <flash_utils.h>
 #include <MD5Builder.h>
 #include <functional>
+#include <erase_config.h>
```

Comment:
Arduino.h already includes this file always, now.

Reply:
okay